### DISCUSSION TITLE
Fleet authentication error 500 forbidden

### DISCUSSION THREAD

**Discussion by Leche671** [Security-Onion-Solutions/securityonion#8345] [Unsupported Versions]
hello, I've run into an issue with Fleet on my securityonion manager node (virtualized on proxmox).

2.3.120
on-prem
iso install
single manager with 4 Search nodes
1 Forwarder node
no issues seeing network traffic.
so-status shows all container statuses as OK

salt-call state-highstate
No errors

no failures on Grid

I can not log into Fleet. it worked for a few hours after install of manager, but now I receive 500 forbidden. Chrome and Firefox

If I add a new user via terminal, I receive:

so-user-add user@....com
Enter new password:
Syncing users and roles between SOC and Elastic...
Elastic state will be re-applied to affected minions. This may take several minutes...
Successfully added new user to SOC
Successfully added user to TheHive
Unable to add user to Fleet; Fleet Service account login failed
Login failed: login received status 500 forbidden: forbidden

Seems like same exact issue as #7002 but not if the fix actions resolved my issue either.

**Comment by ernstdieter** [comment]
Hi,
check the logs on the fleet server.
I had a similar  issue: to many connections to database. I think the default value was 151. I extend it and fixed the authentication issue on Fleet.

**Reply by Leche671** [reply]
Where do you make those changes.

**Comment by m021998** [comment]
From #7002 , I ended up blowing away that entire installation and reinstalling from scratch,
